src/test/regress/expected/multi_mx_call.out

```diff
@@ -27,366 +43,252 @@ select create_distributed_table('mx_call_dist_table_enum', 'key');
 (1 row)
 
 insert into mx_call_dist_table_enum values (1,'S'),(2,'A'),(3,'D'),(4,'F');
-CREATE PROCEDURE mx_call_dist_object(procname text, tablerelid regclass, argument_index int) LANGUAGE plpgsql AS $$
-BEGIN
-    update citus.pg_dist_object
-    set distribution_argument_index = argument_index, colocationid = pg_dist_partition.colocationid
-    from pg_proc, pg_dist_partition
-    where proname = procname and oid = objid and pg_dist_partition.logicalrelid = tablerelid;
-END;$$;
-CREATE PROCEDURE mx_call_proc(x int, INOUT y int) LANGUAGE plpgsql AS $$
+CREATE PROCEDURE mx_call_proc(x int, INOUT y int)
+LANGUAGE plpgsql AS $$
 BEGIN
+    -- groupid is 0 in coordinator and non-zero in workers, so by using it here
+    -- we make sure the procedure is being executed in the worker.
     y := x + (select case groupid when 0 then 1 else 0 end from pg_dist_local_group);
-    y := y + (select sum(t1.val + t2.val) from mx_call_dist_table t1 join mx_call_dist_table2 t2 on t1.id = t2.id);
+    -- we also make sure that we can run distributed queries in the procedures
+    -- that are routed to the workers.
+    y := y + (select sum(t1.val + t2.val) from mx_call_dist_table_1 t1 join mx_call_dist_table_2 t2 on t1.id = t2.id);
 END;$$;
-CREATE PROCEDURE mx_call_proc_asdf(INOUT x mx_call_enum, INOUT y mx_call_enum) LANGUAGE plpgsql AS $$
+-- create another procedure which verifies:
+-- 1. we work fine with multiple return columns
+-- 2. we work find in combinaiton with custom types
```

Comment:
combination

Reply:
Fixed in the sql file, upstream is failing to build, so cannot generate the out file.